automatic/gitextensions/gitextensions.nuspec

```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0"?>
+<package xmlns="http://schemas.microsoft.com/packaging/2011/08/nuspec.xsd">
+  <metadata>
+    <id>gitextensions</id>
+    <version>2.51.02</version>
+    <title>Git Extensions (Install)</title>
+    <authors>Henk Westhuis</authors>
+    <owners>chocolatey</owners>
+    <licenseUrl>https://github.com/gitextensions/gitextensions/blob/master/LICENSE.md</licenseUrl>
+    <projectUrl>http://gitextensions.github.io/</projectUrl>
+    <iconUrl>https://cdn.rawgit.com/ferventcoder/chocolatey-packages/02c21bebe5abb495a56747cbb9b4b5415c933fc0/icons/gitextensions.svg</iconUrl>
+    <requireLicenseAcceptance>false</requireLicenseAcceptance>
+    <description>Git Extensions is the only graphical user interface for Git that allows you control Git without using the commandline. It comes with a manual and video tutorials to get you started quickly.</description>
+    <summary>Git Extensions is the only graphical user interface for Git that allows you control Git without using the commandline.</summary>
+    <releaseNotes>https://github.com/gitextensions/gitextensions/blob/master/GitUI/Resources/ChangeLog.md</releaseNotes>
+    <tags>gitextensions git gui admin</tags>
```

Comment:
legal folder missing

Reply:
there's a legal folder. you mean the license file? if so, why is that needed when there's a licenseUrl element in nuspec?

Reply:
Both tools and legal folder must be added in nuspec. License file is needed because chocolatey gallery requires it. 

See https://github.com/chocolatey/chocolatey-coreteampackages/blob/master/CONTRIBUTING.md#114-embed-a-package-if-allowed